dev/MenuBar/MenuBar_TestUI/MenuBarPage.xaml.cs

```diff
@@ -91,5 +91,16 @@ private void TestFrameCheckbox_Unchecked(object sender, RoutedEventArgs e)
             var testFrame = Window.Current.Content as TestFrame;
             testFrame.ChangeBarVisibility(Visibility.Collapsed);
         }
+
+        private void AddMenuBarToEmptyMenuBarItem_Click(object sender, RoutedEventArgs e)
```

Comment:
can you please add an interaction test ?

Reply:
Of course, missed that.

Reply:
New interaction test added now